.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -62,7 +62,10 @@ schedules:
 # Global variables
 variables:
   buildConfiguration: Release
-  dotnetCoreSdk5Version: 5.0.401
+  dotnetCoreSdk6Version: 6.0.100-rc.2.21505.57
```

Comment:
I think we can merge `dotnetCoreSdk6Version` and `dotnetCoreSdkLatestVersion`? Just use `dotnetCoreSdkLatestVersion`?

Reply:
Replaced in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/d61803a9a585de7d00c45a525bea1341bd2d5c4b